wgs.yaml

```diff
@@ -518,5 +518,56 @@ workinggroups:
   - name: xgboost-operator
     owners:
     - https://raw.githubusercontent.com/kubeflow/xgboost-operator/master/OWNERS
+- dir: wg-data
```

Comment:
@andreyvelich do we still need this definition "here", considering this already exists in internal-acls instead? 🤔 

https://github.com/kubeflow/internal-acls/blob/b4531448425e762736db66a8c123e2edb9e6e9b9/github-orgs/kubeflow/org.yaml#L1056

Reply:
@tarilabs Actually, this file is used to auto-generate README using this generator script: https://github.com/kubeflow/community/tree/master/generator.
It doesn't have any relationship with GitHub teams.

Reply:
Thank you for your feedback!

Should I update these entries with the ones from the gh teams yaml?

Reply:
Yes, let's update them. And re-run the generator script.
Also, when you have time, please take a look at the comments that I left 6 months ago, so we can merge the initial charter for WG Data.

Reply:
Yes, per https://github.com/kubeflow/community/pull/673#issuecomment-2208760477 this is at least in mine todo list, now that the pressing from 1.9 are secured. Thank you for your patience.